script/DeploymentGlobals.sol

```diff
@@ -6,8 +6,14 @@ import "../src/V4/PeanutV4.sol";
 
 contract DeploymentGlobals {
     mapping(uint256 => address) public ecoAddressByChain;
+    address public squidAddress;
 
     constructor() {
+        // SET SQUID
+        squidAddress = 0xce16F69375520ab01377ce7B88f5BA8C48F8D666;
```

Comment:
Yeah, but idk how to reliably detect a testnet 🤔 

I personally won't be a fan of making a list (e.g.  saying "80001, 1115511, 84531 are testnets and all others aren't") since when we will add a new testnet, it won't be in this list => squid mainnet address will be used there silently => we might end up being confused.

Reply:
yeah making an invidual list by chainId sucks.  We could add it as a property to the current chainIds in foundry.toml though and then read it in deploy.py. But it seems kind of overkill, especially since squid address is constant for all mainnets.

Let's keep it as is for now and just be mindful when deploying to testnets. Can update in the future.